src/webui/www/private/scripts/mocha-init.js

```diff
@@ -979,18 +979,32 @@ const initializeWindows = function() {
         }
     };
 
-    ['pause', 'resume'].each(function(item) {
-        addClickEvent(item + 'All', function(e) {
+    addClickEvent('pauseAll', (e) => {
+        if (confirm('QBT_TR(Would you like to pause all torrents?)QBT_TR[CONTEXT=MainWindow]')) {
             new Event(e).stop();
             new Request({
-                url: 'api/v2/torrents/' + item,
+                url: 'api/v2/torrents/pause',
                 method: 'post',
                 data: {
                     hashes: "all"
                 }
             }).send();
             updateMainData();
-        });
+        }
+    });
+
+    addClickEvent('resumeAll', (e) => {
+        if (confirm('QBT_TR(Would you like to resume all torrents?)QBT_TR[CONTEXT=MainWindow]')) {
+            new Event(e).stop();
```

Comment:
Perhaps the event stopper should occur first? (the same to `pauseAll` event)
```suggestion
        new Event(e).stop();
        if (confirm('QBT_TR(Would you like to resume all torrents?)QBT_TR[CONTEXT=MainWindow]')) {
```

Reply:
Absolutely valid point, fixed it.